### ISSUE TITLE
Roles system file in playbooks directory

### ISSUE THREAD

**Issue by verizonold** [redhat-nfvpe/kube-ansible#254]
@dougbtv @leifmadsen  i think I am having the issue because of the changes to playbooks and roles in kube-ansible. I just got the master branch and it looks like now the roles is a "system" file in playbooks directory.
I have to modify multus.yaml.j2 since it uses eth0 and interface i need configured is eth1. How is the roles.system file in playbooks directory updated by kube-ansible install?

**Comment by dougbtv** [comment]
Hey @verizonold did you per chance make sure to run the galaxy requirements install?

```
ansible-galaxy install -r requirements.yml
```

**Comment by verizonold** [comment]
yes, I did after I did a git clone. Shall I run it again and try? Not sure it has to be re-run.

**Comment by dougbtv** [comment]
Ahhh, I believe this is a symlink...

```
[doug@localhost playbooks]$ ls -la | grep -i roles
lrwxrwxrwx  1 doug doug    8 Aug 20 22:20 roles -> ../roles
```
